### PR TITLE
Tag SolveDSGE.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [RJDennis/SolveDSGE.jl](https://github.com/RJDennis/SolveDSGE.jl)
Release: [v0.1.0](https://github.com/RJDennis/SolveDSGE.jl/releases/tag/v0.1.0)
Diff: [vs v0.0.1](https://github.com/RJDennis/SolveDSGE.jl/compare/d2f33b803cadc534be4259319e76f8b7d77a2c5a...8dacd5ccd7f1042c3b9ae2416d1e9f4fe7c23248)
`requires` vs v0.0.1: 
```diff
--- 0.0.1/requires
+++ 0.1.0/requires
@@ -1 +1 @@
-julia 0.3
+julia 0.5.2
```
cc: @RJDennis

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: SolveDSGE/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.5.2
```
Comment (tkelman) [inline SolveDSGE/versions/0.1.0/requires]: is there something in 0.5.2 that wasn't in 0.5.0 that you're relying on?

**Comment by tkelman** [comment]
Doesn't look like you've actually turned Travis CI on, see https://travis-ci.org/RJDennis/SolveDSGE.jl

File: SolveDSGE/versions/0.1.0/requires (hunk 1 above)
Comment (RJDennis) [inline SolveDSGE/versions/0.1.0/requires]: I've changed this to julia 0.5.0.

**Comment by RJDennis** [comment]
Travis is now turned on.  Thanks for your patience.

File: SolveDSGE/versions/0.1.0/requires (hunk 1 above)
Comment (tkelman) [inline SolveDSGE/versions/0.1.0/requires]: see the attobot readme for instructions on how to amend the tag so attobot will update the pr

**Comment by aviks** [comment]
Hi @RJDennis  Can you update the tag? We can merge this then.

**Comment by KristofferC** [comment]
Rerun CI

**Comment by KristofferC** [comment]
Should put `NLSolve` in `test/REQUIRE` if you depend on it for the tests.

**Comment by RJDennis** [comment]
I have removed the need for NLsolve.  Master passes the Travis CI tests for Julia 0.5 and 0.6.

**Comment by martinholters** [comment]
So this has been superseded by #13113 (tagging v0.1.1) and can be closed? (Note that the v0.1.1 that's supposed to be tagged there has Julia minimum version 0.6.0, so this PR and that one are mutually exclusive at the moment. If this PR was to be merged, bumping the required Julia version further would require a new minor version, i.e. v0.2.0).

**Comment by RJDennis** [comment]
I understood that this PR had already been closed, which is why I opened another.  I have changed the REQUIRE file back to Julia 0.5.2 and it passes tests on both 0.5 and 0.6.  I would prefer for the package to be tagged as v0.1.0 if possible.

**Comment by tkelman** [comment]
unless there's something in julia 0.5.2 that you depend on that wasn't in 0.5.0, you typically should not require a patch version

**Comment by aviks** [comment]
This tag, and the one in #13113 are different (ie, they refer to different git commits). Also while this one is for julia `v0.5`, that one is for `0.6` only. Not sure what needs to happen here.

**Comment by RJDennis** [comment]
So I should change the REQUIRE file to Julia 0.5?

**Comment by martinholters** [comment]
Ok, the latest commit to your Repo ([b2f0275](https://github.com/RJDennis/SolveDSGE.jl/tree/b2f0275e2ac03ad15d1cc4286588275b26a08f59)) looks like a good candidate for being tagged. Now there's two options:
* If you want to preserve your existing git tags (which is often recommended, but usually not that important for Julia packages), you just tag as v0.1.2. We close this PR and #13113, and v0.1.0 and v0.1.1 exists in your repo, but not in METADATA, which I think is not a problem.
* Or you delete the v0.1.1 tag and move the v0.1.0 tag to the new commit, and attobot will close #13133 (I think, otherwise we do manually) and update this PR. This is the option I recommend.

**Comment by RJDennis** [comment]
I ended up going with the first option as I wasn't sure how to update the existing PR.

**Comment by martinholters** [comment]
Ok. Then this has been superseded by #13205 now.

**Pull request closed without merging**